lib/web/websocket/frame.js

```diff
@@ -2,26 +2,40 @@
 
 const { maxUnsigned16Bit } = require('./constants')
 
+const BUFFER_SIZE = 16386
+
 /** @type {import('crypto')} */
 let crypto
+const buffer = Buffer.allocUnsafe(BUFFER_SIZE)
```

Comment:
I would rather not allocate 16KB even if WebSocket is never used.

Reply:
@tsctx 

You dont need to allocate here. this can be initialized with null anyway, because if bufIdx === BUFFER_SIZE you will anyway randomFill it in generateMask. 

```suggestion
const buffer = null
```

Reply:
@Uzlopak that doesn't work because randomFillSync expects a buffer (etc.) as its first argument.